src/snake.rs

```diff
@@ -25,6 +21,7 @@ fn setup(mut commands: Commands) {
         .insert(Head)
         .insert(Location::default())
         .insert(Moving::default())
+        // FIXME: this bundle should be in render (somehow)!
```

Comment:
This sprite bundle should really be part of render, but you need to associate it with the entity when the snake is spawned.

Reply:
With bevy, Is there some way to make a trigger on entity creation? ie. `render` plugin could watch for `snake::Head` creations and add the bundle.

Reply:
Ok did this with Added<snake::Head>, will be 1 frame late, but not a problem.

Reply:
That's clever! I didn't know that `Added<>` was a thing. I tried to do that exact thing with a startup system that's `.after(snake::setup)`, and queries the entity `With<snake::Head>`, but the entity wouldn't have spawned in the same stage.